src/ImageSharp.Web/DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -67,7 +67,8 @@ public static IImageSharpBuilder AddImageSharp(this IServiceCollection services)
 
             builder.AddProcessor<ResizeWebProcessor>()
                    .AddProcessor<FormatWebProcessor>()
-                   .AddProcessor<BackgroundColorWebProcessor>();
+                   .AddProcessor<BackgroundColorWebProcessor>()
+                   .AddProcessor<JpegQualityWebProcessor>();
```

Comment:
Should this really be on by default or is it slightly more niche and thus opt in?

Reply:
I think people will expect it. I know it's commonly used in the old ImageProcessor libraries.

Reply:
sounds good then.